### PR TITLE
Starting 2.1.0 Release

### PR DESCRIPTION
**Pull request by v-eacaoi**
## Description

Starting Dotnet 2.1.0 SDK release

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,41 +1,59 @@
 # App Center SDK for .NET Change Log
 
-## Version 2.1.0 (Under development)
+## Version 2.1.0
 
 ### App Center
 
 #### iOS
 
+* **[Fix]** Remove Keychain permission pop-up on macOS.
```
Comment (guperrot) [inline CHANGELOG.md]: We do not support MacOS on Xamarin, please remove this line.
```suggestion
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,41 +1,59 @@
...
 * **[Fix]** Improve encryption security.
 
-### App Center Crashes
+#### Android
```
Comment (guperrot) [inline CHANGELOG.md]:
```suggestion
#### Android

```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,41 +1,59 @@
...
+* **[Fix]** Handle incorrect usage of `AppCenter.setLogUrl` API to provide readable error message.
+* **[Fix]** Fix decrypting values that have been stored for more than a year (such as the in-app update token).
+
+### App Center Analytics
 
 #### iOS
+* **[Feature]** Support setting latency of sending events via `[MSAnalytics setTransmissionInterval:]`.
```
Comment (guperrot) [inline CHANGELOG.md]: Please roll back that as we don't support it in Xamarin, same for android

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,41 +1,59 @@
...
+
+#### Android
+* **[Feature]** Support setting latency of sending events via `Analytics.setTransmissionInterval`.
 
+### App Center Crashes
+
+#### iOS
 * **[Fix]** Fix a crash when tracking an exception without a message.
 
+#### Android
```
Comment (guperrot) [inline CHANGELOG.md]:
```suggestion
#### Android

```

File: CHANGELOG.md (hunk 3 above)
Comment (v-eacaoi) [reply CHANGELOG.md]: Deleted

**Pull request merged**